fetch/range/resources/long-wav.py

```diff
@@ -118,7 +118,7 @@ def main(request, response):
     bytes_remaining_to_send -= len(initial_write)
 
     while bytes_remaining_to_send > 0:
-        if not response.writer.flush():
+        if not response.writer.write(b"."):
```

Comment:
This seems wrong (it's altering the content of the stream). We should test the return value of `response.writer.write(to_send)` at the end.

Reply:
Sorry, I misread #24617. Code updated :)